src/controller/auth.py

```diff
@@ -0,0 +1,8 @@
+from fastapi_sqlalchemy import db
+import requests
+
+
+def auth_account(id_token: str):
+    auth_url = "https://oauth2.googleapis.com/tokeninfo?id_token="
+    res = requests.get(f"{auth_url}{id_token}")
+    return res.json()
```

Comment:
```suggestion
    return res.json()

```
line ending 빼먹지 말고 넣어주기 :D